src/app/js/main.js

```diff
@@ -8,23 +8,23 @@ function toggleLog() {
   log.style.display = log.style.display === 'none' ? 'block' : 'none';
 }
 
-async function loadConfiguration(){
+async function loadConfiguration() {
   const config = await fetch("../TestHarnessConfig.JSON");
   return await config.json();
 }
 function toggleError() {
   const log = document.getElementById('error-container')
   log.style.display = log.style.display === 'none' ? 'block' : 'none'
 }
-function updateError(err, level = "Critical!"){
+function updateError(err, level = "Critical!") {
   const errLog = document.getElementById('error-container')
   let errMessage = $($.find(".error-message")[0]);
   let errLevel = val = $($.find(".error-level")[0]);
-  if(err){
+  if (err) {
     errMessage.text(err);
-  }else{
-    errMessage.text( '&emsp;Oops, something went wrong!' + '<br/>'
-    + '&emsp;Please ensure Cytoscape application is running and try again. Click Log button for details.')
+  } else {
+    errMessage.text('Oops, something went wrong!' + '<br/>'
```

Comment:
remove this '<br/>' this shouldnt be displayed in the string.

Reply:
This is fixed. Thanks